### DISCUSSION TITLE
A section for completed books

### DISCUSSION THREAD

**Discussion by lukethehawk** [advplyr/audiobookshelf#1560] [Ideas]
Is possible to see all the books I've finished? In stats I can see how many but I don't have a list with the titles.
If not, could it be added to the software?

thank you

**Comment by benonymity** [comment]
There are actually a few existing ways to see the books you've completed; you can go to the library page, and filter by progress>finished, or if you are using the admin account you can look at your account's name in the users page and all the items highlighted in green you've finished.

**Reply by lukethehawk** [reply]
I didn't know about the filter, thank you so much. It's pretty simple this way
